domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt

```diff
@@ -280,6 +283,56 @@ class ProfileManagementController @Inject constructor(
       })
   }
 
+  /**
+   * Updates the download and updates access of an admin profile only on wifi.
```

Comment:
`Updates whether to allow download and update only on wifi or not for admin settings.`

Reply:
Done.